dashboard/src/shared/Config.ts

```diff
@@ -9,6 +9,14 @@ export default class Config {
   public static async getConfig() {
     const url = Config.APIEndpoint;
     const { data } = await axios.get<IConfig>(url);
+
+    // Development environment config overrides
+    // TODO(miguel) Rename env variable to TELEPRESENCE_CONTAINER_NAMESPACE
+    // and remove package.json yarn run mapping once create-react-app is ejected
+    if (process.env.NODE_ENV !== "production" && process.env.REACT_APP_KUBEAPPS_NS) {
```

Comment:
should we put the env var `NODE_ENV=production` in the chart?

Reply:
@andresmgot it's helpful to always think of this as an nginx server serving static HTML when in production, I think you keep thinking of this as an nodejs server ;)